barbican/templates/db-maintain-configmap.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: db-maintain-barbican
+  labels:
+    system: openstack
+    type: configuration
+    component: database
+
+data:
+  db-maintaine.sh: |
```

Comment:
its a typo, but largely irrelevant since we execute all scripts in the entrypoint-maintain.d folder - but I fix